embulk-output-sqlserver/src/main/java/org/embulk/output/SQLServerOutputPlugin.java

```diff
@@ -68,6 +71,40 @@
         public InsertMethod getInsertMethod();
     }
 
+    private static class UrlAndProperties {
+        private final String url;
+        private final Properties props;
+
+        public UrlAndProperties(String url, Properties props)
+        {
+            this.url = url;
+            this.props = props;
+        }
+
+        public String getUrl()
+        {
+            return this.url;
+        }
+
+        public Properties getProps()
+        {
+            return this.props;
+        }
+
+        public Properties getPropsWithMaskedSecret()
```

Comment:
Moved masking code as this: https://github.com/embulk/embulk-output-jdbc/pull/121/commits/905597e0b2db0a1a5acd858b51b69da6e4dcd8c6